jest.server.config.js

```diff
@@ -1,6 +1,22 @@
 // For a detailed explanation regarding each configuration property, visit:
 // https://jestjs.io/docs/en/configuration.html
 
+const packageJson = require('./package.json')
+
+const serverTypesFiles = (types) =>
+  types
+    ? types
+        .split(',')
+        .map((st) => `**/*spec.server.${st.replace(/\s/g, '')}.[j|t]s?(x)`)
+    : ['**/*spec.server.[j|t]s?(x)']
+
+// TEST_SERVER_TYPES should be set as Github secret and contain a comma separated string with server types (viya,sas9,sasjs)
+let testServerTypes = process.env.TEST_SERVER_TYPES
+
+if (!testServerTypes) testServerTypes = packageJson.testServerTypes
+
+testServerTypes = serverTypesFiles(testServerTypes)
```

Comment:
we can write this as `const testServerTypes = serverTypesFiles(process.env.TEST_SERVER_TYPES ?? packageJson.testServerTypes)`

Reply:
makes perfect sense!